fingerprint/infra/bio-sdk/src/main/java/com/simprints/fingerprint/infra/biosdk/NECBioSdkWrapper.kt

```diff
@@ -1,36 +1,60 @@
 package com.simprints.fingerprint.infra.biosdk
 
-import com.simprints.core.ExcludedFromGeneratedTestCoverageReports
+import com.simprints.fingerprint.infra.basebiosdk.FingerprintBioSdk
+import com.simprints.fingerprint.infra.basebiosdk.acquisition.domain.TemplateResponse
+import com.simprints.fingerprint.infra.basebiosdk.acquisition.domain.toDomain
 import com.simprints.fingerprint.infra.basebiosdk.matching.domain.FingerprintIdentity
 import com.simprints.fingerprint.infra.basebiosdk.matching.domain.MatchResult
-import com.simprints.fingerprint.infra.scanner.domain.fingerprint.AcquireFingerprintImageResponse
+import com.simprints.fingerprint.infra.necsdkimpl.acquisition.template.FingerprintTemplateAcquisitionSettings
+import com.simprints.fingerprint.infra.necsdkimpl.acquisition.template.FingerprintTemplateMetadata
+import com.simprints.fingerprint.infra.necsdkimpl.matching.NecMatchingSettings
 import com.simprints.fingerprint.infra.scanner.domain.fingerprint.AcquireFingerprintTemplateResponse
+import com.simprints.fingerprint.infra.scanner.v2.domain.main.message.un20.models.Dpi
+import javax.inject.Inject
+
+class NECBioSdkWrapper @Inject constructor(
+    private val bioSdk: FingerprintBioSdk<Unit,
```

Comment:
Something in this formatting feels really weird. Maybe we could either do all params on new lines or all on the same line instead.